custom-elements/CustomElementRegistry.html

```diff
@@ -623,6 +623,18 @@
     assert_equals(customElements.get('existing-custom-element'), ExistingCustomElement);
 }, 'customElements.get return the constructor of the entry with the given name when there is a matching entry.');
 
+test(function () {
+  assert_equals(customElements.getName(class extends HTMLElement {}), undefined);
+}, 'customElements.getName must return undefined when the registry does not contain an entry with the given constructor');
+
+test(function () {
+    class ExistingCustomElement extends HTMLElement {};
+    assert_equals(customElements.getName(ExistingCustomElement), undefined);
+    customElements.define('existing-custom-element', ExistingCustomElement);
```

Comment:
This will throw because 'existing-custom-element' had already been defined in line 623. Use another name like 'other-custom-element'